extensions/sql-migration/src/wizard/skuRecommendation/skuRecommendationPage.ts

```diff
@@ -630,12 +556,6 @@ export class SKURecommendationPage extends MigrationWizardPage {
 			}
 
 			const errors: string[] = [];
-			if (this._rbg.selectedCardId === undefined || this._rbg.selectedCardId === '') {
-				errors.push(constants.SELECT_TARGET_TO_CONTINUE);
-			}
-			if (this.migrationStateModel._databasesForMigration.length === 0) {
-				errors.push(constants.SELECT_DATABASE_TO_MIGRATE);
-			}
```

Comment:
You can leave these changes. Since I am removing this _rbg card also. I will remove these things not required. Also if you need anything to passed from SkuRecommendation page, we can discuss that.

Reply:
These are not changes related to rgb cards. It is the below view/edit container. I have to remove this container as according to current UI, user can't move to next page if he/she doesn't select databases here. For Assessment results page to come directly without any conditions, this is needed to be removed.

Reply:
There is no need to pass anything from SKU recommendation page. For Recommended config, we will have a common function in the utils or somewhere. Rest all the data is populated.